### ISSUE TITLE
AcquireReleaseTest#testWithCleanupStoreFilesWithNoDataAcquireAndRelease is slightly flaky

### ISSUE THREAD

**Issue by alamar** [OpenHFT/Chronicle-Queue#984]
https://teamcity.chronicle.software/buildConfiguration/Chronicle_ChronicleQueue_SnapshotLinux64/639931?showRootCauses=false&expandBuildChangesSection=true&expandBuildProblemsSection=true&expandBuildTestsSection=true

fails in IDE every now and then:
```
java.lang.AssertionError: 
Expected :1
Actual   :0
<Click to see difference>

	at org.junit.Assert.fail(Assert.java:89)
	at org.junit.Assert.failNotEquals(Assert.java:835)
	at org.junit.Assert.assertEquals(Assert.java:647)
	at org.junit.Assert.assertEquals(Assert.java:633)
	at net.openhft.chronicle.queue.AcquireReleaseTest.testWithCleanupStoreFilesWithNoDataAcquireAndRelease(AcquireReleaseTest.java:117)
```

**Comment by RobAustin** [comment]
Ran 100 times, repeat until failure and it worked

![image](https://user-images.githubusercontent.com/6506056/167157294-e29315d6-7c4e-4831-8480-b4e86e1a6eb0.png)

please reopen if its still an issue.

**Comment by hft-team-city** [comment]
Released in [Chronicle-Queue-5.23ea7](https://github.com/OpenHFT/Chronicle-Queue/releases/chronicle-queue-5.23ea7), [BOM-2.23ea42](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.23ea42)
